docs/developer/bulk-orders.mdx

```diff
@@ -0,0 +1,727 @@
+                  },
+                  {
+                    "amount": {
+                      "amount": 120.0
+                    },
+                    "type": "AUTHORIZATION_SUCCESS"
+                  }
+                ]
+              }
+            ],
+            "invoices": [
+              {
+                "number": "01/12/2020/TEST",
+                "url": "http://www.example.com"
+              }
+            ],
+            "discounts": [
+              {
+                "valueType": "FIXED",
+                "value": 10.0,
+                "reason": "Black Friday"
+              }
+            ]
+          },
+          "errors": []
+        }
+      ]
+    }
+  },
+  "extensions": {
+    "cost": {
+      "requestedQueryCost": 22,
+      "maximumAvailable": 50000
+    }
+  }
+}
+```
+
+### Permission
+
+Since this operation is more powerful than regular order management, it is not automatically assigned to staff users or apps with `MANAGE_ORDERS` permission. The mutation requires a new permission:` MANAGE_ORDERS_IMPORT`.
+
+### Input details
+
+#### User (`OrderBulkCreateUserInput`)
+
+To find a user, the mutation accepts one of the following identifiers: `id`, `email` or `external_reference`. If the user associated with an order doesn’t exist in Saleor database, user's email must be provided as a reference in the order instance.
+
+#### Delivery method (`OrderBulkCreateDeliveryMethodInput`)
+
+Orders in Saleor can be shipped to customers or collected directly from warehouse. To identify the type of delivery either `warehouseId` or `shippingMethodId` must be provided. There can also be scenario, when an order doesn’t require physical delivery. It happens when all order’s lines have `isShippingRequired` flag set to `false`. In this case, `OrderBulkCreateDeliveryMethodInput` can be omitted.
+
+The input also accepts denormalized names of delivery methods (`warehouseName`and `shippingMethodName` fields) and tax class (`taxClassName`).
+
+If `shippingPrice` is not provided, Saleor will fetch the current shipping method price from database.
+
+#### Order (`OrderBulkCreateInput`)
+
+The mutation allows to set `Order.number` value to any string, as long as it is unique. By default, when new order is created, Saleor generates a number in sequence to store a unique order number. If an integer value is passed to the `number` field, Saleor updates the current value in the underlying database sequence.
+
+#### Order line (`OrderBulkCreateOrderLineInput`)
+
+`totalPrice` and `undiscountedTotalPrice` are the primary sources of truth about the order pricing. Based on the fields, Saleor calculates unit price, undiscounted unit price, unit discount amount, order total and subtotal.
+
+To find product variant, the mutation accepts one of the following identifiers: `id`, `sku` or `external_reference`.
+
+The input also accepts denormalized names of variant (`variantName`), product (`productName`) and tax class (`taxClassName`).
```

Comment:
`The input also accepts denormalized names of variant`

For front-end engineer not familiar with database terminology it might be confusing, perhaps would be more accessible to name it:
`The input also accepts arbitrary names of variant`